applications/vanilla/views/modules/usercomments.php

```diff
@@ -0,0 +1,33 @@
+<?php if (!defined('APPLICATION')) exit();
+$user = $this->data('User');
+echo '<div class="DataListWrap">';
+echo '<h2 class="H">'.t('Recent Comments').'</h2>';
+echo '<ul class="DataList SearchResults">';
```

Comment:
When you're outputting consecutive chunks of HTML, like this, it's best to avoid outputting them with PHP code. Example:

```php
<?php if (!defined('APPLICATION')) exit();
$user = $this->data('User');
?>
<div class="DataListWrap">
    <h2 class="H"><?php echo t('Recent Comments'); ?></h2>
    <ul class="DataList SearchResults">
```